cards/src/main/resources/cards/custom/group2/set_633/token_shitakiri_slit_tongue_suzume.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Shitakiri, Slit-Tongue Suzume",
+  "baseManaCost": 10,
+  "type": "MINION",
+  "heroClass": "JADE",
+  "baseAttack": 9,
+  "baseHp": 6,
+  "rarity": "FREE",
+  "race": "ELEMENTAL",
+  "description": "Deflect. Your other minions have Deflect during your opponent's turn.",
+  "aura": {
```

Comment:
This should probably be a `TurnStartTrigger` + `TurnEndTrigger` because deflect can be lost normally and incorrectly regained due to board changes when it's an attribute aura. Make sure to write a test for this. you will use multiple `triggers`. Basically at the start of the opponent's turn give the minions deflect, and at the end of the turn, remove deflect only if it's a minion that didn't already have deflect at the start of the turn. Kind of complicated.